build/centos7-puppet6/Puppetfile

```diff
@@ -44,7 +44,9 @@
 #   └── saz-sudo (v6.0.0)
 forge "https://forgeapi.puppetlabs.com"
 
-mod 'ghoneycutt-facter'
+# Current bug: https://github.com/ghoneycutt/puppet-module-facter/issues/70
```

Comment:
Can you describe this error in more detail too? The upstream issue seems incomplete from its description.

Reply:
@rush-skills i added a bunch more detail to the error. Just haven't had time to get back to this.